assets/css/room-modal.css

```diff
@@ -0,0 +1,42 @@
+.hidden {
+  display: none;
+}
```

Comment:
isn't there a Tailwind CSS class for that?

Reply:
added in the last version, where I also added the page and the modal